command/delegate/delegate.go

```diff
@@ -424,13 +403,10 @@ func (c *delegateCommand) handleDestroy(poolManager *vmpool.Manager) http.Handle
 
 		fmt.Printf("\n\nExecuting cleanup: %v\n", reqData)
 
-		pool := poolManager.Get(reqData.Pool)
-		instance := &vmpool.Instance{
-			ID: reqData.ID,
-			IP: "", // TODO remove this
-		}
-		destroyErr := pool.Destroy(r.Context(), instance)
-		if destroyErr != nil {
+		ctx := r.Context()
```

Comment:
nit no need for this line.

Reply:
how's that? the `ctx` is used just below.

Reply:
why instantiate a variable, when you can just call the method on line 408